lib/Db/SourceMapper.php

```diff
@@ -51,6 +51,25 @@ public function find(int|string $id): Source
 		return $this->findEntity(query: $qb);
 	}
 
+	/**
+	 * Find all sources that belong to a specific reference.
+	 *
+	 * @param string $reference The reference to find sources for
+	 * @return array<Source> Array of Source entities
+	 */
+	public function findByRef(string $reference): array
```

Comment:
@rubenvdlinde ik kan op OpenConnector geen Sources meer importeren omdat de findByRef function op de SourceMapper niet meer bestaat
Is dit intentional? Of mag ik deze functie terug zetten?

<img width="1387" height="569" alt="image" src="https://github.com/user-attachments/assets/e02d418e-08aa-411a-bbab-cdc8a437cca8" />
<img width="1185" height="1033" alt="image" src="https://github.com/user-attachments/assets/d8f227a4-8c11-40ab-9953-03df77db6242" />